src/config/ember.ts

```diff
@@ -1,7 +0,0 @@
-import Ember from 'ember';
-
-export default Ember;
-
-const tracked = Ember._tracked;
```

Comment:
We had it because ember has internal kinda `wrapper` over `glimmer/tracking`.

Inputs: 
https://github.com/emberjs/ember.js/tree/master/packages/%40glimmer/tracking
https://github.com/emberjs/ember.js/blob/master/packages/%40ember/-internals/metal/lib/tracked.ts

Reply:
Oops, revert it.

Reply:
Should Ember be imported from `config/ember` or module `ember`?

Reply:
in most cases from 'ember`, `config/ember` used is resolution alias for `@glimmer/tracking`